src/worldofzuul/Environment.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+import java.util.HashMap;
+
+/**
+ *
+ * @author ViktoriaNadarajah
+ */
+public class Environment {
+    private HashMap<String, Room> rooms;
```

Comment:
Missing `docblock` comment. 😄

Reply:
Both attributes and methods.

Reply:
fixed in: 514c5e8cab8545e8bab67a5dde687ac8426441fa